src/SB/Game/zNPCMessenger.h

```diff
@@ -10,23 +10,26 @@ void zNPCMsg_Startup();
 void zNPCMsg_Shutdown();
 void zNPCMsg_AreaNotify(zNPCCommon* sender, en_NPC_MSG_ID msgid, F32 rad, S32 filter,
                         en_NPCTYPES toNPCType);
+void zNPCMsg_Timestep(xScene* xscn, F32 dt);
 
-struct NPCPSClt {
+struct NPCPSClt
+{
     // total size: 0x10
-    void (* notify)(void *, class NPCMsg *); // offset 0x0, size 0x4
-    void * notedata; // offset 0x4, size 0x4
+    void (*notify)(void*, class NPCMsg*); // offset 0x0, size 0x4
```

Comment:
Class keyword is unnecessary artifact, I'm assuming from DWARF data. Please remove